### PR TITLE
[SPARKTA-392] Log files are too big and spend a lot of hard drive resources

### PR DESCRIPTION
**Pull request by anistal**
#### Description
Log files are too big and spend a lot of hard drive resources

### CODE REVIEW

**Comment by compae** [comment]
LGTM

**Comment by danielcsant** [comment]
Could you disable the Spark logs?

**Comment by danielcsant** [comment]
spark logs are still showing

File: dist/src/main/resources/log4j.xml [hunk 1]
```diff
@@ -16,21 +16,51 @@
...
+        <level value="OFF" />
+    </logger>
+    <logger name="log4j.logger.org.apache.avro.ipc.NettyTransceiver">
+        <level value="OFF" />
+    </logger>
+    <logger name="log4j.logger.org.apache.hadoop">
+        <level value="OFF" />
+    </logger>
+    <logger name="log4j.logger.org.apache.spark">
+        <level value="OFF" />
```
Comment (danielcsant) [inline dist/src/main/resources/log4j.xml]: Please, instead of completely disable it, set level to ERROR in every logger

**Comment by compae** [comment]
LGTM

**Comment by danielcsant** [comment]
spark logs are still showing in my laptop

**Comment by danielcsant** [comment]
Duplicated in #1428

**Pull request closed without merging**